hedera-dependency-versions/build.gradle.kts

```diff
@@ -226,4 +226,6 @@ dependencies.constraints {
 
     api("com.google.protobuf:protoc:3.25.4")
     api("io.grpc:protoc-gen-grpc-java:1.66.0")
+    api("com.hedera.cryptography:hedera-cryptography-pairings-signatures:0.1.0-SNAPSHOT")
+    api("com.hedera.cryptography:hedera-cryptography-tss:0.1.0-SNAPSHOT")
```

Comment:
I would recommend against this.
We had this approach before (things on hedera-services depending on snapshot builds of hedera-cryptography), but I removed it; snapshot dependencies can break already cutoff release branches.

Reply:
Removed snapshot dependency and temporarily copied the needed types.